monai/networks/utils.py

```diff
@@ -93,9 +93,17 @@ def one_hot(labels: torch.Tensor, num_classes: int, dtype: torch.dtype = torch.f
     return labels
 
 
-def slice_channels(tensor: torch.Tensor, *slicevals: Optional[int]) -> torch.Tensor:
+def slice_channels(tensor: torch.Tensor, dim: int = 1, *slicevals: int) -> torch.Tensor:
```

Comment:
this looks very generic, could be used to slice any single dimension and works for tensor/ndarray, so I think the function name and type annotations should be updated to be more generic...

Reply:
Sure, makes sense to me.
I updated the function and args name / type and moved to utils in the latest commit.
Could you please help review it again?

Thanks in advance.